build/pack-docs.ps1

```diff
@@ -8,7 +8,7 @@
 #>
 
 param(
-
+    [string] $Version
```

Comment:
At this point, it's not even a version number, but a Docker image tag. Renaming the parameter accordingly to make it clear that it is directly used as an image tag may help alleviate future confusion.

Reply:
Agreed, perhaps something like `TagSuffix`?

Reply:
Thanks @cgranade . I just did, and the change did a reset on your vote. :)